mcs/class/corlib/System.Reflection.Emit/TypeBuilder.cs

```diff
@@ -120,8 +120,15 @@ protected override TypeAttributes GetAttributeFlagsImpl ()
 			return attrs;
 		}
 
+		TypeBuilder ()
```

Comment:
This will have to be manually ported to dotnet/runtime

Reply:
This is just for normal mono, netcore mono has different code.